views/home.tmpl

```diff
@@ -186,73 +186,185 @@
-                        <span style="flex: 0 0 50px" class="lh1rem text-right ml-1">Size</span>
-                        <span style="flex: 0 0 62px" class="lh1rem text-right ml-1 jsonly">Age</span>
-                    </div>
-                    <div data-target="homepage.transactions" class="transactions md-height-rows rows">
-                    {{range .Mempool.LatestTransactions}}
-                        <div class="d-flex flex-table-row mempool-row">
-                            <a class="hash truncate-hash keyboard-target" style="flex: 1 1 auto" href="/tx/{{.Hash}}" title="{{.Hash}}">{{.Hash}}</a>
-                            <span style="flex: 0 0 65px" class="mono text-right ml-1">{{.Type}}</span>
-                            <span style="flex: 0 0 105px" class="mono text-right ml-1 fs13">
-                              {{template "decimalParts" (float64AsDecimalParts .TotalOut 8 false)}}
-                            </span>
-                            <span style="flex: 0 0 50px" class="mono text-right ml-1">{{.Size}} B</span>
-                            <span style="flex: 0 0 62px" class="mono text-right ml-1 jsonly" data-target="time.age" data-age="{{.Time}}"></span>
-                        </div>
-                    {{end}}
-                    </div>
+                {{with .BestBlock}}
+                <div class="d-flex justify-content-between align-items-center px-2 mx-auto">
+                  <div class="text-left mt-2 d-inline-block">
+                    <span class="h6 mb-0 position-relative" data-tooltip="stakeholders are validating this block now.">proposed</span><br>
+                    <a class="d-inline-block h3 position-relative" href="/block/{{.Hash}}" data-target="homepage.blockHeight">{{.Height}}</a>
+                  </div>
+                  <div class="text-right my-2 d-inline-block">
+                    <span class="h6 position-relative" data-tooltip="total DCR sent in this block">value</span><br>
+                    <span class="h4" data-target="homepage.blockTotal">{{threeSigFigs .Total}}</span><span class="h6"> DCR</span>
+                  </div>
                 </div>
 
-                <div class="d-flex align-items-center">
-                    <h4>Latest Blocks</h4> <a href="/blocks" class="pl-2 keyboard-target"><small>see more ...</small></a>
+                <div class="d-flex justify-content-between align-items-center px-2 mx-auto">
+                    <div class="text-left mt-2 pr-3 d-inline-block">
+                      <span class="h6 d-inline-block mb-2 position-relative" data-tooltip="stakeholder's votes for the proposed block.">votes</span><br>
+                      <div class="d-inline-block p1 h4" data-target="homepage.blockVotes" data-hash="{{.Hash}}">
+                        {{range $index, $vote := $.BlockTally}}
+                          <span class="d-inline-block dcricon-{{if eq $vote 0}}reject{{else if eq $vote 1}}affirm{{else}}missing{{end}}"></span>
```

Comment:
I think each of `dcricon-{reject,affirm,missing}` deserve tooltips.  Especially in the missing case.  Perhaps for missing it could be something like "this selected ticket has yet to vote" or "waiting for this vote".  The affirm and reject options are much clearer, but I personally think it's worth reiterating whenever we get the chance that the votes is to validate/invalidate the regular transaction tree for this block.